File: lab2/multi.java
import java.util.ArrayList;

public class Main {
    public static void main(String[] args) {
        ArrayList<ArrayList<String>> nestedList = new ArrayList<>();
        
        // Добавляем элементы в списки
        nestedList.add(new ArrayList<>(List.of("apple", "banana")));
        nestedList.add(new ArrayList<>(List.of("cherry", "grape")));
        nestedList.add(new ArrayList<>(List.of("mango", "kiwi")));
        
        System.out.println(nestedList.get(0).get(1));  // выводит 'banana'
    }
}

File: lab2/multi.py
nested_list = [
    ["apple", "banana"],
    ["cherry", "grape"],
    ["mango", "kiwi"]
]

flat_list = []

# Проходим циклом по каждому внутреннему списку и добавляем элементы в flat_list
for inner_list in nested_list:
    for item in inner_list:
        flat_list.append(item)

print(flat_list)  # ['apple', 'banana', 'cherry', 'grape', 'mango', 'kiwi']